### PR TITLE
Change context menu color to blue/red

### PR DESCRIPTION
**Pull request by yuxitang-amzn**
*Issue #, if available:*

*Description of changes:*
Change context menu button text color from black to blue (normal) and red (delete).

Screenshots:
|Before|After|
|--|--|
|image|image|
|image|image|
|image|image|
|image|image|

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by yuxitang-amzn** [comment]
Open discussion here:
1. ContextMenu accepts a children of arbitrary `ReactElement[]` and in each page creates an list of `EuiButtonEmpty` (current solution). In this way we have an capability to support other component (like toggle button).
2. ContextMenu accepts `EuiButtonEmptyProps` and render `EuiButtonEmpty` inside. In this way we have an capability to update the style of all buttons together.

File: public/apps/configuration/panels/role-list.tsx [hunk 1]
```diff
@@ -138,17 +136,18 @@ export function RoleList(props: AppDependencies) {
...
       key="edit"
       onClick={() => {
         window.location.href = buildHashUrl(ResourceType.roles, Action.edit, selection[0].roleName);
       }}
       disabled={selection.length !== 1 || selection[0].reserved}
     >
       Edit
-    </EuiContextMenuItem>,
+    </EuiButtonEmpty>,
+    // <EuiSpacer size='xs' />,
```
Comment (tianleh) [inline public/apps/configuration/panels/role-list.tsx:148]: remove commented code.

File: public/apps/configuration/panels/role-list.tsx [hunk 2]
```diff
@@ -160,27 +159,19 @@ export function RoleList(props: AppDependencies) {
       disabled={selection.length !== 1}
     >
       Duplicate
-    </EuiContextMenuItem>,
-    <EuiContextMenuItem
+    </EuiButtonEmpty>,
+    // <EuiSpacer size='xs' />,
```
Comment (tianleh) [inline public/apps/configuration/panels/role-list.tsx:163]: remove commented code.

**Comment by yuxitang-amzn** [comment]
> Did we agreed to have horizontal lines to separate the options in the drop down menu?

No we didn't introduce horizontal rule for the drop on listing page. See the screenshot, and below is the mockup. None of them have hr.
image

File: public/apps/configuration/panels/role-list.tsx (hunk 1 above)
Comment (yuxitang-amzn) [reply public/apps/configuration/panels/role-list.tsx]: Sure removed.

File: public/apps/configuration/panels/role-list.tsx (hunk 2 above)
Comment (yuxitang-amzn) [reply public/apps/configuration/panels/role-list.tsx]: Sure removed.

**Comment by tianleh** [comment]
> Open discussion here:
> 
>     1. ContextMenu accepts a children of arbitrary `ReactElement[]` and in each page creates an list of `EuiButtonEmpty` (current solution). In this way we have an capability to support other component (like toggle button).
> 
>     2. ContextMenu accepts `EuiButtonEmptyProps` and render `EuiButtonEmpty` inside. In this way we have an capability to update the style of all buttons together.

Currently there is no such use case for supporting more types. Also I doubt we will have such toggle button support in the future. Personally I would prefer option 2. But I am also OK with option 1. I will leave it to you to make the call.

**Comment by yuxitang-amzn** [comment]
> > Open discussion here:
> > ```
> > 1. ContextMenu accepts a children of arbitrary `ReactElement[]` and in each page creates an list of `EuiButtonEmpty` (current solution). In this way we have an capability to support other component (like toggle button).
> > 
> > 2. ContextMenu accepts `EuiButtonEmptyProps` and render `EuiButtonEmpty` inside. In this way we have an capability to update the style of all buttons together.
> > ```
> 
> Currently there is no such use case for supporting more types. Also I doubt we will have such toggle button support in the future. Personally I would prefer option 2. But I am also OK with option 1. I will leave it to you to make the call.

You're right, while as option 1 is already implemented. Let's not predicting the future and use it first. We can have another PR switching to option 2 if needed.

**Comment by bandinib-amzn** [comment]
> Open discussion here:
> 
> 1. ContextMenu accepts a children of arbitrary `ReactElement[]` and in each page creates an list of `EuiButtonEmpty` (current solution). In this way we have an capability to support other component (like toggle button).
> 2. ContextMenu accepts `EuiButtonEmptyProps` and render `EuiButtonEmpty` inside. In this way we have an capability to update the style of all buttons together.

Second option looks more neat. But for now 1st approach would work. Also in future need to check if we can support icon in ContextMenu

**Pull request merged**